modules/installation-supported-aws-machine-types.adoc

```diff
@@ -29,6 +29,11 @@ endif::[]
 
 The following Amazon Web Services (AWS) instance types are supported with {product-title}.
 
+[NOTE]
+====
+Use the machine types included in the following charts for your instances. If you use an instance type that is not listed in the chart, ensure that the instance size you use matches the minimum resource requirements of the listed instance type. 
```

Comment:
> matches the minimum resource requirements of the listed instance type. 

`minimum resource requirements` are described in the above table, change it to, for example, `minimum resource requirements, see Minimum resource requirements for cluster installation section` ?

Reply:
@yunjiang29 Just updated! Can you take once last look? 
I will update this PR once #49799 is merged. Thank you very much for your help with this PR!